code/modules/mob/living/carbon/human/species/species.dm

```diff
@@ -134,18 +134,20 @@
 /datum/species/proc/create_organs(var/mob/living/carbon/human/H) //Handles creation of mob organs.
 
 	for(var/obj/item/organ/organ in H.contents)
-		if((organ in H.organs) || (organ in H.internal_organs))
+		if((organ in H.organs))
 			qdel(organ)
 
+	//for(var/obj/item/organ/internal/organ in H.contents)
+	//	if((organ in H.internal_organs))
+	//		qdel(organ)
+
 	if(H.organs)                  H.organs.Cut()
-	if(H.internal_organs)         H.internal_organs.Cut()
+	//if(H.internal_organs)         H.internal_organs.Cut()
```

Comment:
Perhaps this should look through the list of organs in the mob, and remove all the organs that match the old species and dna?